fathom/src/surface/pretty.rs

```diff
@@ -424,6 +438,151 @@ impl<'interner, 'arena> Context<'interner, 'arena> {
             .group()
         }
     }
+
+    fn pretty_if<Range>(
+        &'arena self,
+        prec: Prec,
+        cond_expr: &Term<'_, Range>,
+        then_expr: &Term<'_, Range>,
+        else_expr: &Term<'_, Range>,
+    ) -> DocBuilder<'arena, Self> {
+        let cond = docs![
+            self,
+            "if",
+            self.space(),
+            self.term_prec(Prec::Fun, cond_expr),
+        ];
```

Comment:
Generally I’ve been using `self.concat([ ... ])` for this. Is it ok if we switch to that for consistency?